modules/nlu/src/backend/language-provider.ts

```diff
@@ -103,6 +103,21 @@ export class RemoteLanguageProvider implements LanguageProvider {
     return this
   }
 
+  private handleLanguageServerError = (err, endpoint: string, logger) => {
+    const status = _.get(err, 'failure.response.status')
+    const details = _.get(err, 'failure.response.message')
+
+    if (status === 429) {
+      logger.error(
+        `Could not load Language Server: ${details}. You may be over the limit for the number of requests allowed for the endpoint ${endpoint}`
+      )
+    } else if (status === 401) {
+      logger.error(`Could not load Language Server. You must a valid authentication token for the endpoint ${endpoint}`)
```

Comment:
Could not load ? what do you mean by load ?

I would remove this and keep the message from "You must **provide** a valid authentication token ..."